plutus-ledger/src/Ledger/Typed/Tx.hs

```diff
@@ -224,10 +205,11 @@ typeScriptTxIn
     -> TxIn
     -> m (TypedScriptTxIn inn)
 typeScriptTxIn lookupRef si TxIn{txInRef,txInType} = do
-    (vs, rs, ds) <- case txInType of
+    (_, rs, ds) <- case txInType of
```

Comment:
you could just make it return `(rs, ds)`

Reply:
:man_facepalming: